pir/pir-internal/src/main/AndroidManifest.xml

```diff
@@ -8,11 +8,18 @@
         <activity
             android:name=".settings.PirResultsActivity"
             android:label="@string/pirDevSimpleScanHeader" />
+
         <service
-            android:name=".scan.PirForegroundScanService"
+            android:name=".service.PirForegroundScanService"
             android:enabled="true"
             android:exported="false"
             android:foregroundServiceType="specialUse"
             android:process=":pir" />
+
+        <service
+            android:name=".service.PirRemoteWorkerService"
+            android:exported="false"
+            android:permission="android.permission.BIND_JOB_SERVICE"
+            android:process=":pir" />
```

Comment:
Workaround since the VPN forces `androidx.work.multiprocess.RemoteWorkerService` to run on vpn service.